taxonomies/ingredients.txt

```diff
@@ -1261,6 +1261,10 @@ sk:Sladidlo
 wikidata:en:Q626292
 #wikidata:en:Q4368298
 
+<en:citric acid
```

Comment:
We are gradually removing additives from the ingredients.txt file because we automatically include the additives.txt file

So we should not add additives to ingredients.txt

Reply:
removed